src/zimwriterfs/zimwriterfs.cpp

```diff
@@ -296,6 +308,21 @@ void parse_args(int argc, char** argv)
       }
     }
   } while (c != -1);
+  
+  if(checkDescriptionLengths()){
+    std::cerr << "Long description should not be shorter than the short description." << std::endl;
+    exit(1);
+  }
+  
+  if (description.length() > 80) {
+    std::cerr << "Description length exceeds the 80 character limit." << std::endl;
+    exit(1);
+  }
+
+  if (!longDescription.empty() && longDescription.length() > 4000) {
+    std::cerr << "Long description length exceeds the 4000 character limit." << std::endl;
+    exit(1);
+  }
```

Comment:
1. This (except the call to `exit()` can be moved inside the `checkDescriptionLengths()` function
2. Please also add a check on description being non-empty (that will fix #336).